src/DbImporter/Exceptions/DatabaseImportFailed.php

```diff
@@ -58,4 +58,12 @@ public static function timeoutExceeded($timeout)
     {
         return new static('The sql process exceeded the timeout limit of '.$timeout.' seconds. This timeout may be tweaked in your config/backup.php file.');
     }
+
+    /**
+     * @return DatabaseImportFailed
+     */
+    public static function notEnoughDiskSpace()
```

Comment:
```suggestion
    public static function insufficientDiskSpace($available, $required)
```

I think it would make sense to have the metrics passed through constructor, instead of having the disk-space logic in the actual exception. 
Maybe also let the message say what is the minimum required disk space to perform the operation.